webpack.config.js

```diff
@@ -0,0 +1,32 @@
+var path = require('path');
+
+module.exports = {
+  entry: __dirname + '/src/main.jsx',
+  output: {
+    path: __dirname + '/build/js',
+    filename: '[name].bundle.js'
+  },
+  externals: {
+    'react': 'React',
```

Comment:
ahh gotcha, cool. we bust that up manually by checking the `NODE_ENV` and conditionally adding those plugins on teach.